Refit.Tests/Verifiers/CSharpSourceGeneratorVerifier`1.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace Refit.Tests
+{
+    public static partial class CSharpSourceGeneratorVerifier<TSourceGenerator>
```

Comment:
📝 This file would normally contain helpers for the most common cases, but we haven't identified those yet for source generators.